variables.tf

```diff
@@ -1,24 +1,28 @@
-variable "authorized_aggregator_account_id" {
-  description = "Account ID where the authorized aggregator exists; required when `create_config_authorization` is `true`"
-  type        = string
-  default     = ""
+variable aggregator {
+  description = "Object specifying the configuration of a Config Aggregator"
+  type = object({
+    name = string
+    tags = map(string)
+    account_aggregation_source = object({
```

Comment:
This is a nit and a bit silly, but it would be nice if there was a similar order to the object member list.  Since I'm new to Terraform, I'm looking for patterns and omissions, so when I compare the initialization of this object to the object below, I'm looking to see if there are similar fields or missing fields (and if those differences matter).  Since the member lists are not in the same order, this takes a teensy more work.  Again, just a nit.

Reply:
That is exactly the kind of thing I am looking for in a review. Good eye! In this case, each object maps to a different configuration block, and those blocks inherently accept different attributes. So the objects are and must be different. What I've done here is just sort the attributes alphabetically to reduce some cognitive burden on choosing and maintaining the order.